Sources/Sentry/Public/SentryScreenFrames.h

```diff
@@ -1,18 +1,45 @@
 #import "SentryDefines.h"
+#import "SentryProfilingConditionals.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 #if SENTRY_HAS_UIKIT
 
+/** An array of dictionaries that each contain a start and end timestamp for a rendered frame. */
+typedef NSArray<NSDictionary<NSString *, NSNumber *> *> SentryFrameInfoTimeSeries;
+
 @interface SentryScreenFrames : NSObject
 SENTRY_NO_INIT
 
 - (instancetype)initWithTotal:(NSUInteger)total frozen:(NSUInteger)frozen slow:(NSUInteger)slow;
 
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+- (instancetype)initWithTotal:(NSUInteger)total
+                       frozen:(NSUInteger)frozen
+                         slow:(NSUInteger)slow
+              frameTimestamps:(SentryFrameInfoTimeSeries *)frameTimestamps
+          frameRateTimestamps:(SentryFrameInfoTimeSeries *)frameRateTimestamps;
+#    endif // SENTRY_TARGET_PROFILING_SUPPORTED
+
 @property (nonatomic, assign, readonly) NSUInteger total;
 @property (nonatomic, assign, readonly) NSUInteger frozen;
 @property (nonatomic, assign, readonly) NSUInteger slow;
 
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+/**
+ * Array of dictionaries describing slow/frozen frames' timestamps. Each dictionary start and end
+ * timestamp for every such frame, keyed under @c start_timestamp and @c end_timestamp.
+ */
+@property (nonatomic, copy, readonly) SentryFrameInfoTimeSeries *frameTimestamps;
```

Comment:
`m`: Wouldn't it maybe be better to have two different properties? One for slow and one for frozen frames? Then we could name them `slowFrameTimestamps` and `frozenFrameTimestamps`. This would improve the readability of the code, IMO.